stripe/test/test_requestor.py

```diff
@@ -40,11 +45,10 @@ def __eq__(self, other):
                 self._extra_match(other))
 
     def _keys_match(self, other):
-        expected_keys = self.EXP_KEYS + self.extra.keys()
+        expected_keys = set(self.EXP_KEYS + self.extra.keys())
```

Comment:
I needed to make this a set because otherwise `Stripe-Version` will appear in one list twice and the results will not match.